python/gtsam/examples/SFMExample_bal.py

```diff
@@ -0,0 +1,119 @@
+"""
+  GTSAM Copyright 2010, Georgia Tech Research Corporation,
+  Atlanta, Georgia 30332-0415
+  All Rights Reserved
+  Authors: Frank Dellaert, et al. (see THANKS for the full author list)
+
+  See LICENSE for the license information
+
+  Solve a structure-from-motion problem from a "Bundle Adjustment in the Large" file
+  Author: Frank Dellaert (Python: Akshay Krishan, John Lambert)
+"""
+
+import argparse
+import logging
+import sys
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+import gtsam
+from gtsam import (
+    GeneralSFMFactorSfmCamera,
```

Comment:
Please format entire file with google style

Reply:
I'm following `black` here for the multi-line imports. Is there a particular PEP number you could point me to for Google style?

Reply:
https://github.com/psf/black